Engage Cockpit CM/CD Documentation + Settings Section (#7790)
umbraco/UmbracoDocs: 17/umbraco-engage/getting-started/for-developers/cockpit.md

![Umbraco Engage Cockpit - Detailed information](../../.gitbook/assets/engage-cockpit-2.png)

If the Cockpit is missing and the Umbraco backoffice runs on a different domain, see the [Load Balancing and CM/CD Environments](loadbalancing-and-cm-cd-environments.md) article.

## Opening the Cockpit in CM/CD Environments

> In load-balanced setups where the backoffice runs on a separate CM server, you can open the Cockpit using the **Open Cockpit** button in the Engage dashboard. This provides secure authentication without requiring cookie domain configuration.

Reviewer: A suggestion to shorten the first sentence:

```suggestion
In load-balanced setups where the backoffice runs separately, you can use the **Open Cockpit** button in the Engage dashboard to open the Cockpit. This provides secure authentication without requiring cookie domain configuration.
```
